modules/delegates/components/DelegatedByAddress.tsx

```diff
@@ -110,6 +112,9 @@ const CollapsableRow = ({ delegator, network, bpi, totalDelegated }: Collapsable
                       parseUnits(lockAmount.indexOf('-') === 0 ? lockAmount.substring(1) : lockAmount)
                     )}${bpi > 0 ? ' MKR' : ''}`}
                   </Text>
+                  <Text key={blockTimestamp} variant="smallCaps" sx={{ pl: 2 }}>
+                    {isLockstake && !!isUser ? '(Seal module)' : ''}
```

Comment:
Why the `!!isUser` check? Shouldn't we show the `(Seal module)` text regardless?